Source/Extensions/Blazorise.DataGrid/_DataGridPagination.razor.cs

```diff
@@ -25,7 +25,7 @@ private PagerElementPosition GetTotalItemsPosition()
             => ParentDataGrid.PagerOptions?.TotalItemsPosition ?? PagerElementPosition.Default;
 
         private Size GetSize()
-            => ParentDataGrid.PagerOptions?.ButtonSize ?? Size.Default;
+            => ParentDataGrid.Theme?.PaginationOptions?.Size ?? ParentDataGrid.PagerOptions?.ButtonSize ?? Size.Default;

```

Comment:
I think we want to have DataGrid ButtonSIze higher priority than themes.

```
=> ParentDataGrid.PagerOptions?.ButtonSize ?? ParentDataGrid.Theme?.PaginationOptions?.Size ?? Size.Default;
```

Reply:
Oh my god you're right. That's more specific, nice catch. Will change.